CMakeLists.txt

```diff
@@ -76,6 +75,41 @@ else()
   message(STATUS "Doxygen generation - Disabled")
 endif()
 
+
+##############################################################################
+# Wheel
+##############################################################################
+
+if (WHEEL_GENERATION)
+  include(FetchContent)
+  message(STATUS "Wheel generation - Enabled")
+  FetchContent_Declare(python-cmake-wheel
+  GIT_REPOSITORY "https://github.com/maliput/python-cmake-wheel"
+  GIT_TAG        "main"
+  GIT_SHALLOW    ON)
+
+  FetchContent_GetProperties(python-cmake-wheel)
+  FetchContent_MakeAvailable(python-cmake-wheel)
+  list(APPEND CMAKE_MODULE_PATH "${python-cmake-wheel_SOURCE_DIR}")
+  include(python-wheel)
+  set(WHEEL_DEPLOY_DIRECTORY "${CMAKE_BINARY_DIR}/wheel")
+
+  set(DEPLOY_FILES "${PROJECT_SOURCE_DIR}/maliput_malidrive/__init__.py")
```

Comment:
Don't we deploy the sample maps?

Reply:
`DEPLOY_FILES` accepts a list of files, I can't add a folder (in this case resources).

https://github.com/maliput/python-cmake-wheel/blob/main/python-wheel.cmake#L124-L131

I could use `resources/*` and that will copy all the files directly which will be a mess because they will be thrown in the wheel.

We could also extend that feature in order to accept folders. However probably it makes sense to leave it as an enhancement issue on top of this PR.

I am also thinking that probably, once that is done, we could add a method in the __init__ file for getting a resource so it is easier to deal with.

Reply:
I think the tooling is lacking the ability to add [data-files](https://docs.python.org/3.8/distutils/setupscript.html#installing-additional-files). Perhaps we can add it in the future. I agree with deferring that to a follow up PR.

Reply:
https://github.com/maliput/maliput_malidrive/issues/250